templates/landingpage.html.j2

```diff
@@ -481,7 +495,12 @@
                             <div class="col-sm-10">
                                 <span>
                                         {% for affiliation in contact.Affiliation %}
-                                            {{ affiliation }}{{ ", " if not loop.last }}
+                                            {% if affiliation.Affiliation_Identifier | length > 0 %}
```

Comment:
This is missing the "affiliation is mapping" condition for backwards compatibility?